openssl/goopenssl.h

```diff
@@ -726,6 +734,7 @@ DEFINEFUNCINTERNAL(int, RSA_pkey_ctx_ctrl,
 		   (EVP_PKEY_CTX *ctx, int optype, int cmd, int p1, void *p2),
 		   (ctx, optype, cmd, p1, p2))
 
+    #include <openssl/rsa.h>
```

Comment:
This is already include at https://github.com/golang-fips/openssl-fips/pull/10/files#diff-fd756b4114a2dc406db5c6fe6e0aa83892568d7536d6944decc24be096657067R479

Reply:
Done.